pom.xml

```diff
@@ -325,6 +333,19 @@
 	        <artifactId>springdoc-openapi-ui</artifactId>
 	        <version>1.5.9</version>
         </dependency>
+        <dependency>
```

Comment:
Do you need following dependencies? And if yes, for which functionality

Reply:
I have not added this dependency as it was present before.